docs/getting-started/setup.md

```diff
@@ -16,6 +16,17 @@ The first tool we will be installing is [`cargo-contract`](https://github.com/pa
 
 As a pre-requisite for the tool you need to install the [binaryen](https://github.com/WebAssembly/binaryen) package, which is used to optimize the WebAssembly bytecode of the contract.
 
+For example, on Ubuntu or Debian, run the following command:
+
+```bash
+sudo apt install binaryen
+```
```

Comment:
The latest stable [binaryen release is `v99`](https://tracker.debian.org/pkg/binaryen) which is too old for us to use, so this actually doesn't work at the moment. However, it is still a valid way of installing `wasm-opt`, so we can keep it but can you add a [caution admonition](https://docusaurus.io/docs/markdown-features/admonitions
) right below the installation instruction mentioning that users should manually download an up to date binary instead?